remind.py

```diff
@@ -395,16 +395,19 @@ def to_remind(self, vevent, label=None, priority=None, tags=None):
         elif isinstance(trigdates, str):
             remind.append(trigdates)
 
+        def abbr_tag(tag):
```

Comment:
Could you add a `@staticmethod`? Also, what about adding an _ to the method name, as it's rather internal.